doc/NPCs.md

```diff
@@ -1410,21 +1354,65 @@ _some functions support array arguments or kwargs, denoted with square brackets
 | weather(`s`)  |  ✅  |   ✅   | N/A<br/>(global)  | Return or set a weather aspect<br/><br/>Aspect must be one of:<br/>`temperature` (in Kelvin),<br/>`humidity` (as percentage),<br/>`pressure` (in millibar),<br/>`windpower` (in mph).<br/>`precipitation` (in mm / h) either 0.5 (very_light ), 1.5 (light), or 3 (heavy). Read only.<br/><br/>Temperature conversion functions are available: `celsius()`, `fahrenheit()`, `from_celsius()`, and `from_fahrenheit()`.<br/><br/>Examples:<br/>`{ "math": [ "weather('temperature')", "<", "from_fahrenheit( 33 )" ] }`<br/>`{ "math": [ "fahrenheit( weather('temperature') )", "==", "21" ] }`|
 | damage_level()    |  ✅   |   ❌  | u, n  | Return the damage level of the talker, which must be an item.<br/><br/>Example:<br/>`"condition": { "math": [ "n_damage_level()", "<", "1" ] }`|
 
-More examples:
+#### List of Character and item aspects
+These can be read or written to with `val()`.
+
+| Value | Supports assignment | Description |
+--- | --- | --- |
+| `activity_level` | ❌ | Current activity level index as a floored integer, from 0-5. Roughly: <pre>0.45 = SLEEP_EXERCISE (floored and returns 0),<br/>0.5 = NO_EXERCISE(floored and returns 0),<br/>1 = LIGHT_EXERCISE,<br/>2 = MODERATE_EXERCISE,<br/>3 = BRISK_EXERCISE,<br/>4 = ACTIVE_EXERCISE,<br/>5 = EXTRA_EXERCISE. |
+| `age` | ✅ | Current age in years. |
+| `allies` | ❌ | The avatar's number of allies |
+| `anger` | ✅ | Current anger level. Only works for monsters. |
+| `bmi_permil` | ❌ | Current BMI per mille (Body Mass Index x 1000) |
+| `body_temp` | ❌ | Current body temperature. |
+| `body_temp_delta` | ❌ | Difference in temperature between the hottest/coldest part and what feels like the hottest/coldest part. |
+| `cash` | ❌ | Amount of money |
+| `dodge` | ❌ | Current effective dodge |
+| `exp` | ✅ | Total experience earned. |
+| `fatigue` | ✅ | Current fatigue level. |
+| `fine_detail_vision_mod` | ❌ | Returned values range from 1.0 (unimpeded vision) to 11.0 (totally blind). |
+| `focus` | ✅ | Current focus level. |
+| `friendly` | ✅ | Current friendly level. Only works for monsters. |
+| `grab_strength` | ❌ | Grab strength as defined in the monster definition. Only works for monsters |
+| `health` | ❌ | Current health level. |
+| `height` | ✅ | Current height in cm. When setting there is a range for your character size category. Setting it too high or low will use the limit instead. For tiny its 58, and 87. For small its 88 and 144. For medium its 145 and 200. For large its 201 and 250. For huge its 251 and 320. |
+| `hunger` | ❌ | Current perceived hunger. |
+| `instant_thirst` | ❌ | Current thirst minus water in the stomach that hasn't been absorbed by the body yet. |
+| `mana` | ✅ | Current mana. |
+| `mana_max` | ❌ | Max mana. |
```

Comment:
```suggestion
| `mana_max` | ❌ | Max mana. |
| `mana_percentage` | ❌ | Current mana as percent. |
```
Something i was about to document, but forgot to do

Reply:
Btw, I plan to redo the documentation and follow the format you've been using for EOC elsewhere. The current table is a big failure as it's basically unreadable in raw format and can't have TOC links.